cmd/security-secrets-setup/Dockerfile

```diff
@@ -44,25 +44,22 @@ COPY . .
 
 RUN make cmd/security-secrets-setup/security-secrets-setup
 
-FROM vault:1.0.2
+FROM alpine:latest
 
 LABEL license='SPDX-License-Identifier: Apache-2.0' \
-      copyright='Copyright (c) 2019: Dell Technologies, Inc.'
+      copyright='Copyright (c) 2019: Dell Technologies, Inc.  Copyright (c) 2019 Intel Corporation'
 
 USER root
 
 # install necessary tools
-RUN apk update && apk add ca-certificates dumb-init jq \
+RUN apk update && apk add ca-certificates dumb-init \
```

Comment:
Please add the boilerplate mentioned in this issue: https://github.com/edgexfoundry/edgex-go/issues/2130

Reply:
i 'll address that in a separate PR